packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2883,6 +2885,19 @@ class _Location {
   }
 }
 
+/// Returns the if an object is user created.
+///
+/// This function will only work in debug mode builds when
+/// the `--track-widget-creation` flag is passed to `flutter_tool`. Dart 2.0 is
+/// required as injecting creation locations requires a
+/// [Dart Kernel Transformer](https://github.com/dart-lang/sdk/wiki/Kernel-Documentation).
+bool isUserCreated(Object object) {
```

Comment:
can we call this isLocalCreationLocation
for consistency with the existing name?
I'm also open to changing the existing name as it is private if there is a clear case this name is better.

Reply:
I think it is fine to use the existing name

Reply:
fixed